### PR TITLE
enhance configuration sample, extends prov app cm.yml with default openshift permissioned groups

### PR DESCRIPTION
**Pull request by stitakis**
completes ProvApp feature https://github.com/opendevstack/ods-provisioning-app/pull/636

### CODE REVIEW

**Comment by stitakis** [comment]
@borja44 fyi

**Comment by stitakis** [comment]
@felipecruz91 problem again with github build and go... can you please have a look?

File: configuration-sample/ods-core.env.sample [hunk 1]
```diff
@@ -298,6 +298,9 @@ PROV_APP_PROVISION_CLEANUP_INCOMPLETE_PROJECTS_ENABLED=false
 # List of jira project templates
 JIRA_PROJECT_TEMPLATE_KEY_NAMES=${project.template.default.key},kanban
 
+# List of default groups to be permissioned in openshift on project creation (value is optional and can be empty)
```
Comment (felipecruz91) [inline configuration-sample/ods-core.env.sample:301]: @stitakis Could you include a comment line with an example of how it would be used, please?

**Review by felipecruz91 — CHANGES_REQUESTED** [review]

File: configuration-sample/ods-core.env.sample (hunk 1 above)
Comment (stitakis) [reply configuration-sample/ods-core.env.sample:301]: something like this?
```
#DEFAULT_OPENSHIFT_PERMISSION_GROUPS="ADMINGROUP=MY-DEFAULT-CLUSTER-ADMIN-GROUP"
```

Comment (felipecruz91) [reply configuration-sample/ods-core.env.sample]: That's perfect, thanks @stitakis

**Review by felipecruz91 — APPROVED** [review]

**Review by michaelsauter — APPROVED** [review]

**Pull request merged**